src/esp/gfx/GenericDrawable.cpp

```diff
@@ -139,8 +139,12 @@ void GenericDrawable::updateShaderLightingParameters(
 
   for (Mn::UnsignedInt i = 0; i < lightSetup_->size(); ++i) {
     const auto& lightInfo = (*lightSetup_)[i];
-    lightPositions.emplace_back(getLightPositionRelativeToCamera(
-        lightInfo, transformationMatrix, cameraMatrix));
+    Mn::Vector4 pos = getLightPositionRelativeToCamera(
+        lightInfo, transformationMatrix, cameraMatrix);
+    // flip directional lights to faciliate faster, non-forking calc in
+    // shader.  Leave non-directional lights unchanged
+    pos *= (pos[3] * 2) - 1;
+    lightPositions.emplace_back(pos);
```

Comment:
```suggestion
    lightPositions.emplace_back(std::move(pos));
```

Reply:
Oh whoops, I think it actually has a trivial copy ctor. It is some  ofthe Eigen classes are movable.